canal/sync.go

```diff
@@ -5,12 +5,18 @@ import (
 
 	"golang.org/x/net/context"
 
+	"regexp"
+
 	"github.com/juju/errors"
 	"github.com/ngaut/log"
 	"github.com/siddontang/go-mysql/mysql"
 	"github.com/siddontang/go-mysql/replication"
 )
 
+var (
+	expAlterTable = regexp.MustCompile("^ALTER TABLE\\s.*?`([^`\\.]+?)`\\s.*")
```

Comment:
Hi @iyidan 

Could you add some test for this, like 
```
alter table a.b
alter table `a.b`
alter table b
alter table `b`
```

we can all get table `b` here.